.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2
```

Comment:
Please move the comment from the previous circle config starting from ` # Setup in CircleCI account the following ENV variables: `

Might save time for someone as it's not clear at all which env variables are required for this build to work.